kasa/tests/fixtures/KL430(US)_1.0_1.0.10.json

```diff
@@ -40,29 +40,68 @@
             "is_factory": false,
             "is_variable_color_temp": 1,
             "latitude_i": 0,
-            "length": 16,
+            "length": 14,
             "light_state": {
-                "brightness": 50,
-                "color_temp": 3630,
+                "brightness": 15,
+                "color_temp": 2500,
                 "hue": 0,
                 "mode": "normal",
                 "on_off": 1,
                 "saturation": 0
             },
             "lighting_effect_state": {
-                "brightness": 50,
+                "brightness": 100,
                 "custom": 0,
                 "enable": 0,
-                "id": "",
-                "name": ""
+                "id": "bCTItKETDFfrKANolgldxfgOakaarARs",
+                "name": "Flicker"
             },
             "longitude_i": 0,
-            "mic_mac": "CC32E5230F55",
+            "mic_mac": "CC32E5000000",
             "mic_type": "IOT.SMARTBULB",
             "model": "KL430(US)",
             "oemId": "00000000000000000000000000000000",
-            "preferred_state": [],
-            "rssi": -56,
+            "preferred_state": [
+                {
+                    "brightness": 100,
+                    "custom": 0,
+                    "id": "QglBhMShPHUAuxLqzNEefFrGiJwahOmz",
+                    "index": 0,
+                    "mode": 2
+                },
```

Comment:
I mix&matched these two fixture files to gather as much information as possible.